docs/maps/api-reference.md

```diff
@@ -235,3 +235,27 @@ mySound.play(config);
 // ...
 mySound.stop();
 ```
+
+
+### register additional menu entries
+
+adds an additional Entry to the main menu , these exist until the map is unloaded
+
+
+```typescript
+registerMenuCommand(menuCommand: string, callback: (menuCommand: string) => void): void
+```
+Example:
+
+
+```javascript
+
+WA.registerMenuCommand("test", () => {
+    WA.sendChatMessage("test clicked", "menu cmd")
+})
+
+```
+
+<div class="col">
```

Comment:
i  hope this actually renders in the checked in version 🤔

Reply:
https://github.com/jonnytest1/workadventuremap/blob/master/test/markdowntest.md working exampel with relative urls

Reply:
It does work in GitHub, but we use the same Markdown file in the workadventu.re website, and I know for a fact it will fail there :(